kolibri_instant_schools_plugin/assets/src/views/sign-in-page/index.vue

```diff
@@ -375,7 +377,10 @@
     text-align: left
     margin: auto
 
-  .login-btn
+  .accent-button
+    $vf-accent-button()
+
+  .login-button
```

Comment:
class got renamed which is why it's not displayed properly anymore

Reply:
d'oh